catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/FilterNodeMapImpl.java

```diff
@@ -100,6 +102,26 @@ public String getValue() {
     return Objects.toString(json.get(VALUE));
   }
 
+  @Override
+  public Double getDistance() {
+    if (!isLeaf()) {
+      throw new IllegalStateException("No target value exists for a logical operator");
+    }
+
+    Object distance = json.get(DISTANCE);
+    if (distance == null) {
+      return null;
+    } else if (distance instanceof Double) {
```

Comment:
✏️ This conditional block could be replaced with one check for `distance instanceof Number`

```
else if (distance instanceof Number) {
    return ((Number) distance).doubleValue();
}
```

Reply:
Worked like a charm! Nice catch